### ISSUE TITLE
make:entity Test --with-ulid or make:entity Test --with-ulid does not use Ulid/Uuid

### ISSUE THREAD

**Issue by lpotherat** [symfony/maker-bundle#1577]
Hi there, I love using Uuids / Ulids as primaries, and I jumped on this feature ! 
But it looks like it works only if we don't pass any parameters `php bin/console make:entity --with-uuid`

**How to reproduce :** 

Execute `php bin/console make:entity Test --with-uuid` ( or `php bin/console make:entity Test --with-ulid` )
Follow steps to class creation and quit.

**Expected :** 
The id field created like this : 

```php
#[ORM\Entity(repositoryClass: TestRepository::class)]
class Test
{
    #[ORM\Id]
    #[ORM\Column(type: UuidType::NAME, unique: true)]
    #[ORM\GeneratedValue(strategy: 'CUSTOM')]
    #[ORM\CustomIdGenerator(class: 'doctrine.uuid_generator')]
    private ?Uuid $id = null;

    public function getId(): ?Uuid
    {
        return $this->id;
    }
}
```

**Actual result:** 

```php
#[ORM\Entity(repositoryClass: TestRepository::class)]
class Test
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    public function getId(): ?int
    {
        return $this->id;
    }
}
```

**Comment by Bisb** [comment]
This occurs when the `symfony/uid` package is not installed. It triggers an error when executing `bin/console make:entity --with-uuid` without parameters, but not when specifying the entity name (e.g., `bin/console make:entity --with-uuid Test`).

For consistency and clarity, the latter command should probably also trigger the same error (`[ERROR] You must install symfony/uid to use UUIDs as "id" (composer require symfony/uid)`).

**Comment by lpotherat** [comment]
Hello Bisb, 

The uid package is correctly installed, and if it was not very clear, it works like a charm when you do not specify the entity.
I pushed a test repository here so it is easy to test : 
https://github.com/lpotherat/symfony-maker-ulid-test 
The uid package is installed and the bug is still there.

I think the consistency bug is not related to the mentionned issue, but it can for sure be addressed 👍

**Comment by Bisb** [comment]
@lpotherat 

You're right, I realize now that I misunderstood the last part of your post.

It seems the root of the problem is the same, the options `--with-uuid` / `--with-ulid` are simply ignored when providing the class name.
